packages/code/src/CopyButton/CopyButton.spec.tsx

```diff
@@ -54,9 +88,21 @@ describe('CopyButton', () => {
       });
     });
 
-    test('closes tooltip when clicking out of focused button', async () => {
+    test('invokes onCopy and copies contents to clipboard onClick', () => {
       const onCopy = jest.fn();
-      const { queryByTestId } = renderCopyButton({ onCopy });
+      const { getByTestId } = renderCopyButton({ onCopy });
+      const copyButton = getByTestId(testIds.copyButton);
+
+      userEvent.click(copyButton);
+      expect(onCopy).toHaveBeenCalledTimes(1);
+      expect(ClipboardJS).toHaveBeenCalledWith(copyButton, {
+        text: expect.any(Function),
```

Comment:
What does this mean?

Reply:
@shaneeza this tests that when copy button is clicked, a new clipboard object is made